src/components/search/empty-state.tsx

```diff
@@ -0,0 +1,29 @@
+import Image from 'next/image';
+
+import { getURLPath } from '../../utils/format-url-path';
+import { thumbnailLoader } from '../card/utils';
+import { TypographyScale } from '@mdb/flora';
+
+const EmptyState: React.FunctionComponent = () => (
+    <div
+        sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'center',
+            alignItems: 'center',
+        }}
+    >
+        <div>
```

Comment:
Why do we need this div here?

Reply:
I think I was having trouble getting the `Image` component to actually center without the `div`. I think there may be some styling on that component making that difficult.